graph/types/twitter_user_info.graphqls

```diff
@@ -0,0 +1,12 @@
+type TwitterUserInfo {
+    id: ID!
+    name: String!
+    diplayName: String!
+    profileImageURL: String!
+    isVerified: Boolean!
+    isInvited: Boolean!
```

Comment:
It makes sense that the invitation field is part of the Twitter info for a user. However, if this is not a desiderable design, I can refactor this in a new "TwitterUserInvitationInfo" type that has an "isInvited" field and a "twitterUserInfo" field.

Reply:
I think with graphql you want to use `invited` instead of `isInvited`